File: server/filters.js
'use strict';

const fs = require('fs');

let rawdata = fs.readFileSync('./server/data/filters.json');
let filters = JSON.parse(rawdata);

module.exports = {
    getJobType    : getJobType,
    getWorkSchedule : getWorkSchedule,
    getExperience    : getExperience,
    getDepartment    : getDepartment
  };

function getJobType() {
    return(filters.job_type);
};

function getWorkSchedule() {
    return(filters.work_schedule);
}

function getExperience() {
    return(filters.experience);
}

function getDepartment() {
    return(filters.department);
}

File: healthexplore.api/healthexplore.filters.api/Models/Filters.cs
namespace healthexplore.filters.api.Models
{
    using System.Collections.Generic;
    using Newtonsoft.Json;

    public class Filters
    {
        [JsonProperty("job_type")]
        public List<JobType> JobType { get; set; }

        [JsonProperty("work_schedule")]
        public List<WorkSchedule> WorkSchedule { get; set; }

        [JsonProperty("experience")]
        public List<Experience> Experience { get; set; }

        [JsonProperty("department")]
        public List<Department> Department { get; set; }
    }

    public class JobType : BaseEntity { }
    public class WorkSchedule : BaseEntity { }
    public class Experience : BaseEntity { }
    public class Department : BaseEntity { }
}